src/amo/components/AutoSearchInput/index.js

```diff
@@ -162,10 +163,20 @@ export class AutoSearchInputBase extends React.Component<Props, State> {
       return;
     }
 
+    if (value.length < SEARCH_TERM_MIN_LENGTH) {
+      log.debug(oneLine`Ignoring suggestions fetch because query
+      subceeds min length (${SEARCH_TERM_MIN_LENGTH})`);
```

Comment:
I know that subceed is sometimes used colloquially but it threw me off at first. I think it would be better to spell it out as "Ignoring ... because query does not meet the required length (${SEARCH_TERM_MIN_LENGTH})"